<!DOCTYPE html>
<html>

<head>

  <title>xi_单细胞空间转录组实战</title>
  <base href="../">
  <meta id="root-path" root-path="../">

  <link rel="icon" sizes="96x96" href="https://publish-01.obsidian.md/access/f786db9fac45774fa4f0d8112e232d67/favicon-96x96.png">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
  <meta charset="UTF-8">
  <script src="https://code.iconify.design/iconify-icon/1.0.3/iconify-icon.min.js"></script>

  <link rel="stylesheet" href="lib/styles/obsidian-styles.css">
  <link rel="stylesheet" href="lib/styles/theme.css">
  <link rel="stylesheet" href="lib/styles/plugin-styles.css">
  <link rel="stylesheet" href="lib/styles/snippets.css">
  <style>
    body.css-settings-manager {
      --h1-color: #FFFFFF;
      --h2-size: 1.8em;
      --h3-color: #83BE6B;
      --h5-color: #FF6943;
      --h6-color: #E83E3E;
      --h4-transform: none;
      --h3-size: 1.6em;
      --h4-size: 1.4em;
      --h5-size: 1.2em;
      --h6-size: 1.0em;
    }

    body.theme-light.css-settings-manager {}

    body.theme-dark.css-settings-manager {}
  </style>


  <script src="lib/scripts/webpage.js"></script>

</head>

<body class="theme-dark mod-windows is-frameless is-hidden-frameless obsidian-app show-inline-title minimal-theme minimal-things-dark minimal-things-light colorful-headings colorful-active system-shade img-grid minimal-folding minimal-readable-off minimal-light minimal-dark chart-default-width table-default-width img-default-width iframe-default-width map-default-width contextual-typography css-settings-manager qe-title-right h2-underline obsidian-themepocalypse is-maximized" style="--zoom-factor:1.44; --font-text-override:Arial, 新宋体; --font-print-override:Arial, 新宋体; --font-text-size:17px; --line-width:1500px; --line-width-adaptive:1500px; --file-line-width:1500px; --content-width:1500px; --sidebar-width:25em; --collapse-arrow-size:0.4em; --tree-horizontal-spacing:1em; --tree-vertical-spacing:0.5em; --sidebar-margin:12px;">
  <div class="webpage-container">
    <div class="sidebar-left sidebar">
      <div class="sidebar-content">
        <div><label class="theme-toggle-container" for="theme_toggle"><input class="theme-toggle-input" type="checkbox" id="theme_toggle">
            <div class="toggle-background"></div>
          </label></div>
      </div>
    </div>
    <div class="document-container">
      <div class="markdown-preview-view markdown-rendered node-insert-event allow-fold-headings show-indentation-guide allow-fold-lists" tabindex="-1" style="tab-size: 4;">
        <style id="MJX-CHTML-styles"></style>
        <div class="markdown-preview-sizer markdown-preview-section" style="padding-bottom: ; padding-top: var(--file-margins); padding-right: var(--file-margins); padding-left: var(--file-margins); width: 100%; position: absolute;">
          <div class="markdown-preview-pusher" style="width: 1px; height: 0.1px; margin-bottom: 0px;"></div>
          <div class="mod-header"></div>
          <div data-tag-name="p" class="el-p">
            <p>测序深度(Sequencing Depth)</p>
          </div>
          <div data-tag-name="ul" class="el-ul">
            <ul class="has-list-bullet">
              <li data-line="0">
                <div class="list-bullet"></div>相当于测了几遍, 指测序得到的总碱基数（bp）与所测物种基因组大小的比值，其是评价测序量的重要指标之一；
              </li>
              <li data-line="1">
                <div class="list-bullet"></div>如：测序产生了1000条读段（read），每条读段的长度为50bp，所测物种的基因组大小为10000bp，那么测序深度为1000 * 50 / 10000=5X, 即读到的总bp数除以基因组bp数
              </li>
              <li data-line="2">
                <div class="list-bullet"></div>一般来说，测序深度越深越好，当然还需考虑一个成本的问题。
              </li>
              <li data-line="3">
                <div class="list-bullet"></div>全基因组测序，一般测序深度为30X以上对检测基因组变异的可靠性会有很大帮助。
              </li>
              <li data-line="4">
                <div class="list-bullet"></div>测序产生的错误率或假阳性结果会随着测序深度的提升而下降。
              </li>
            </ul>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>scRNA-seq process:</p>
          </div>
          <div data-tag-name="ol" class="el-ol">
            <ol>
              <li data-line="0">Isolation of single cells: Individual cells are separated from a heterogeneous sample using methods like fluorescence-activated cell sorting (FACS) or microfluidic devices.</li>
              <li data-line="1">Cell lysis and mRNA capture: The cells are lysed to release their RNA content, and polyadenylated mRNA molecules are captured using oligo-dT primers or other methods.</li>
              <li data-line="2">Reverse transcription and cDNA synthesis: The mRNA molecules are reverse transcribed into complementary DNA (cDNA).</li>
              <li data-line="3">Addition of UMIs and cell barcodes: During library preparation, Unique Molecular Identifiers (UMIs) are added to label individual RNA molecules, and cell-specific barcodes are added to identify the cell of origin for each RNA molecule.</li>
              <li data-line="4">Amplification and library construction: The cDNA molecules are amplified, and adapters are ligated to create a sequencing library.</li>
              <li data-line="5">Sequencing: The library is sequenced using high-throughput sequencing platforms like Illumina, generating short reads.</li>
              <li data-line="6">Data analysis: The reads are aligned to a reference genome or transcriptome, allowing researchers to quantify gene expression levels and identify distinct cell populations, states, and functions.</li>
            </ol>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>read</p>
          </div>
          <div data-tag-name="ul" class="el-ul">
            <ul class="has-list-bullet">
              <li data-line="0">
                <div class="list-bullet"></div>In scRNA-seq (single-cell RNA sequencing), the term "read" refers to the short sequences of nucleotides obtained from the RNA of individual cells during the sequencing process. These reads are generated by a sequencer and represent fragments of the RNA molecules present in each cell. The reads are then aligned to a reference genome or transcriptome, allowing researchers to quantify gene expression levels and identify distinct cell populations, states, and functions within a heterogeneous sample. This technique is particularly valuable for understanding cellular heterogeneity, revealing rare cell types, and studying complex biological processes at the single-cell level.
              </li>
              <li data-line="1">
                <div class="list-bullet"></div>reads是测序得到的一小段碱基序列
              </li>
              <li data-line="2">
                <div class="list-bullet"></div>由于这段序列在测序之前是经过PCR的，所以reads的数量并不能代表RNA的丰度
              </li>
            </ul>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>测序覆盖度（Sequencing Coverage）</p>
          </div>
          <div data-tag-name="ul" class="el-ul">
            <ul class="has-list-bullet">
              <li data-line="0">
                <div class="list-bullet"></div>指测序数据匹配到参考基因组上后，能够覆盖基因组的区域比率。因为基因组有些区域重复序列较复杂或测序难以捕获到，那么这些区域可能就不会被测序读段所覆盖。
              </li>
              <li data-line="1">
                <div class="list-bullet"></div>如测序产生了1000条读段（read），每条读段的长度为50bp，所测物种的基因组大小为10000bp，测序读段匹配到参考基因组后能够覆盖9000bp的参考基因组，那么测序覆盖度就为9000/10000=90%。
              </li>
            </ul>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>从reads到基因名<br>
              统计比对到基因上的reads, 即为的counts, 即为测序原始表达矩阵，rawdata。<br>
              但由于两大原因（不同样本的测序深度，不同基因的长度）直接用counts比较将没有意义，因此需要进行标准化。</p>
          </div>
          <div data-tag-name="ul" class="el-ul">
            <ul class="has-list-bullet">
              <li data-line="0">
                <div class="list-bullet"></div>不同样本的测序深度：例如sample1测序深度为5X，sample2测序深度为15X，自然sample2的基因counts都比sample1的高很多。想要消除，计算比例即可。即针对每一样本，计算每一个基因counts相对于整体测序深度的比例。
              </li>
              <li data-line="1">
                <div class="list-bullet"></div>不同基因的长度：例如geneA长度为1KB，geneB长度为10KB；那么geneB的counts数总体都大于geneA。如果要比较，则计算基因平均每KB的表达量即可。
              </li>
              <li data-line="2">
                <div class="list-bullet"></div>RPKM、FKPM、TPM、CPM为不同的标准化手段
              </li>
            </ul>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>RPKM: Reads Per Kilobase of exon model per Million mapped reads<br>
              RPKM = total exon reads/ (mapped reads (Millions) * exon length (KB))<br>
              每千个碱基的转录每百万映射读取读数。<br>
              公式大致就是某一基因的counts先除以总reads数(反映测序深度)，再除以基因长度。</p>
          </div>
          <div data-tag-name="ol" class="el-ol">
            <ol>
              <li data-line="0">由于总reads太大了，直接除以这个数字就会使得标准化出来的Read数出现太多的小数，所以为了美观，一般都是除以以百万为单位的总Read数；假定一次RNA-seq的总Read数为2 <em> 10^7， 那么在进行Read标准化的时候，并不是直接除以这个数值，而是除以20，因为2 </em> 10^7 = 20 * 10^6 = 20M</li>
              <li data-line="1">第二次除以的基因长度值，按KB为单位，即1000。</li>
            </ol>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>FPKM<br>
              每千个碱基的转录每百万映射读取的片段。</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>TPM<br>
              TPM的计算方法其实也同RPKM很类似，同样的对基因长度和测序深度进行标准化；<br>
              即counts先除基因长度，再除总reads数。这样每个样本最后的结果和都相等，不同样本间差异更清楚。</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>CPM<br>
              即只对测序文库（每个样本总reads数）标准化，而不对长度标准化。这是因为，差异分析往往是同一基因在两组或多组样本量的差异，因此不必在计算单位长度基因的表达量。</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>为什么要单细胞测序：<br>
              bulk RNA seq: 由于每种不同的细胞的RNA不同，如果不考虑细胞类型直接对组织测序，会导致混杂，包括取样的混杂</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>UMIs<br>
              Unique Molecular Identifier<br>
              通过将UMIs纳入测序文库，可以将文库扩增过程中产生的PCR重复序列与真正的生物学重复序列区分开来，并防止高估转录丰度。</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>barcodes<br>
              相同barcode来源于同一个细胞</p>
          </div>
          <div data-is-embed="true" data-embed-type="image" class="el-embed-image el-p" data-tag-name="p">
            <p><span alt="Pasted image 20230319133555.png" src="Pasted image 20230319133555.png" class="internal-embed media-embed image-embed is-loaded"><img alt="Pasted image 20230319133555.png" src="scrna-seq\_attachments\pasted-image-20230319133555.png"></span><br>
              每个spot小室大概包含10-40个细胞</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>DNA seq和RNA seq的差异<br>
              DNA看的是碱基对变异，RNA看的是表达量</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>DNA突变丰度<br>
              突变reads占总reads的百分比<br>
              Reads are aligned to a reference sequence with bioinformatics difference BETWEEN THE RETERENCE Genome and the newly sequenced reads can be identified</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>fastq数据格式，即二代测序的原始数据<br>
              <span alt="Pasted image 20230319163311.png" src="Pasted image 20230319163311.png" class="internal-embed media-embed image-embed is-loaded"><img alt="Pasted image 20230319163311.png" src="scrna-seq\_attachments\pasted-image-20230319163311.png"></span><br>
              第一行: 第一行以@开头的一段编码<br>
              第二行: reads<br>
              第三行: +<br>
              第四行: scores
            </p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>测序读长:reads(rawdata )测序质量<br>
              Q20：每个碱基准确率99%<br>
              Q30：每个碱基准确率99.9%（主要看Q30）<br>
              QC：底线Q20至少&gt;85%，Q30至少&gt;80%</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>scRNA seq的本质：<br>
              一个样本有多少个细胞<br>
              每个细胞的基因表达特征</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>单细胞测序<br>
              Tag-based 3' counting techniques:</p>
          </div>
          <div data-tag-name="ul" class="el-ul">
            <ul class="has-list-bullet">
              <li data-line="0">
                <div class="list-bullet"></div>早期只测3‘端ployA后的一小段基因（&lt;150bps）
              </li>
              <li data-line="1">
                <div class="list-bullet"></div>根据这段基因就可以定位是哪个基因
              </li>
              <li data-line="2">
                <div class="list-bullet"></div>1 read per transcript
              </li>
              <li data-line="3">
                <div class="list-bullet"></div>Expression analysis only
              </li>
              <li data-line="4">
                <div class="list-bullet"></div>Fewer reads per cell needed (-50K reads/cell)
              </li>
              <li data-line="5">
                <div class="list-bullet"></div>Less noise in expression patterns<br>
                Full-length
              </li>
              <li data-line="7">
                <div class="list-bullet"></div>Splicing information
              </li>
              <li data-line="8">
                <div class="list-bullet"></div>The more information desired beyond expression, the higher the reads needed per cell(~50K reads/cell to10M reads/cell)
              </li>
            </ul>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>3’端或者5‘端测序<br>
              均是只测一条RNA的末端或者起始端一段很短的序列测到这一段就能代表表达了一个RNA<br>
              所以3’测序只能提供基本的基因表达量(即一个基因表达了几个RNA)</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>Full-length seq<br>
              则一条RNA完整的被测到<br>
              既能提供基因的表达量(测到几条RNA), 又能知道每条RNA的序列</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>对比:同一条RNA在前者只需要条reads在后者需要很多reads（根据不同的基因长度而不同）</p>
          </div>
          <div data-is-embed="true" data-embed-type="image" class="el-embed-image el-p" data-tag-name="p">
            <p><span alt="Pasted image 20230319173646.png" src="Pasted image 20230319173646.png" class="internal-embed media-embed image-embed is-loaded"><img alt="Pasted image 20230319173646.png" src="scrna-seq\_attachments\pasted-image-20230319173646.png"></span><br>
              <span alt="Pasted image 20230319175055.png" src="Pasted image 20230319175055.png" class="internal-embed media-embed image-embed is-loaded"><img alt="Pasted image 20230319175055.png" src="scrna-seq\_attachments\pasted-image-20230319175055.png"></span>
            </p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>barcode：</p>
          </div>
          <div data-tag-name="ul" class="el-ul">
            <ul class="has-list-bullet">
              <li data-line="0">
                <div class="list-bullet"></div>每一个凝胶微珠都有特定的barcode，不同凝胶微珠上的barcode不同
              </li>
              <li data-line="1">
                <div class="list-bullet"></div>又因为每个细胞只允许与一个凝胶微珠进行结合，所以相同barcode可以用于区分序列是否来自同一个细胞<br>
                UMI：
              </li>
              <li data-line="3">
                <div class="list-bullet"></div>UMI是一段随机序列，紧接着barcode，同一凝胶微珠上的所有UMI都不同
              </li>
              <li data-line="4">
                <div class="list-bullet"></div>UMI起到的作用,是在经过PCR、再深度测序得到的reads，可以看出哪些reads是来自于一个原始cDNA分子的
              </li>
              <li data-line="5">
                <div class="list-bullet"></div>另外PCR对于不同cDNA的效率是不一样的，UMI可以保证对每一个reads溯源到原始的cDNA
              </li>
              <li data-line="6">
                <div class="list-bullet"></div>通过UMI可以对gene的表达进行count，得到单个细胞的表达量<br>
                Poly(dT)
              </li>
              <li data-line="8">
                <div class="list-bullet"></div>用于捕获mRNA的ploy(A)
              </li>
            </ul>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>饱和曲线<br>
              <span alt="Pasted image 20230319200752.png" src="Pasted image 20230319200752.png" class="internal-embed media-embed image-embed is-loaded"><img alt="Pasted image 20230319200752.png" src="scrna-seq\_attachments\pasted-image-20230319200752.png"></span><br>
              原始的reads越多,则被测到的基因数也会越多<br>
              咨询公司以确定需要测量多少reads/cell(但通常用每个样本多少个G表达， v3 kit 10000 cells 大概60-100G)
            </p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>3‘端测序只能测RNA(数据挖掘目前仅限于该领域);5’端测序则实现了RNA+免疫组库的同时测序;<br>
              更高端的还包括:空间转录组测序,定制化建库(通过连接特定抗体蛋白之后只筛选携带特定蛋白的细胞)</p>
          </div>
          <div data-tag-name="p" class="el-p">
            <p>对罕见细胞类型的挖掘</p>
          </div>
          <div class="mod-footer"></div>
        </div>
      </div>
    </div>
    <div class="sidebar-right sidebar">
      <div class="sidebar-content">
        <div class="tree-container outline-tree" data-depth="0">
          <div class="tree-header"><span class="sidebar-section-header">Table Of Contents</span><button class="clickable-icon collapse-tree-button is-collapsed"><iconify-icon icon="ph:arrows-in-line-horizontal-bold" width="18px" height="18px" rotate="90deg" color="currentColor"></iconify-icon></button></div>
          <div class="tree-scroll-area"></div>
        </div>
      </div>
    </div>
  </div>
</body>

</html>